src/MQTT433gateway.cpp

```diff
@@ -236,6 +239,19 @@ void setup() {
     }
   });
 
+  settings.registerChangeHandler(SYSLOG, [](const Settings &s) {
+    if (syslogLog) {
+      Logger.removeHandler(*syslogLog);
+      delete syslogLog;
+    }
+    if (s.syslogLevel.length() != 0 && s.syslogHost.length() != 0 &&
```

Comment:
better use `>` instead of `!=`

Reply:
@janLo any comment here?

Reply:
I think we're safe that the length of a string would not be negative or something else will be very wrong?

Reply:
Also `length()` of `String` is unsigned.

Reply:
Yes, but `> 0` is more secure and nowhere else in the code is `!= 0` used.